c2corg_ui/static/js/apiservice.js

```diff
@@ -310,4 +310,29 @@ app.Api.prototype.updatePreferredLanguage = function(lang) {
 };
 
 
+/**
+ * @return {!angular.$q.Promise<!angular.$http.Response>}
```

Comment:
Being public does not mean they are not or should not be minified. It is an independant concept.